### PR TITLE
Marked settings

### LINKED ISSUE #1: passing options to marked
I want to be able to enable GFM and Tables in marked. It would be great if I could pass options from config into marked. I propose doing this in a similar way to how config options are passed in to dot.js.

### PR DESCRIPTION
**Pull request by DanielLarsenNZ**
Fixes #1. Refactored markdown.js to accept `config.marked_settings` in setup. Added test for table markup.

### CODE REVIEW

**Comment by DanielLarsenNZ** [comment]
@brainshave would you mind reviewing? This is my first time using Jasmine. Any suggestions/improvements welcomed.

File: README.md [hunk 1]
```diff
@@ -115,6 +115,10 @@ line.
 
 Settings for the [doT][dot] template engine.
 
+### marked_settings
+
+Takes any [marked options](https://github.com/chjj/marked/tree/0d27d58f16e3aa478cbc89a9a304e592e2862c5a#options-1).
```
Comment (brainshave) [inline README.md]: Please put the URL in the footer to keep it clean and nice here :-)

File: spec/markdown_spec.js [hunk 2]
```diff
@@ -5,9 +5,13 @@ var trim = require("./trim");
 
 describe("markdown_compiler", function () {
 
-  var compiler = require("../markdown");
+  var markdown = require("../markdown");
+  
+
+
```
Comment (brainshave) [inline spec/markdown_spec.js]: This 3 empty lines are unnecessary.

File: markdown.js [hunk 3]
```diff
@@ -7,17 +7,24 @@
...
+	md.setOptions(settings);
+
+	return {
+	  applies_to: applies_to,
+	  compile: compile
+	};
+
+	function compile (src) {
+	  return md(src);
+	}
```
Comment (brainshave) [inline markdown.js]: It appears as if your editor has set indentation to 8 spaces. All files in this project use 2 spaces.

Comment (DanielLarsenNZ) [inline markdown.js]: bugger. apologies, will fix.

File: spec/markdown_spec.js [hunk 4]
```diff
@@ -5,9 +5,13 @@ var trim = require("./trim");
 
 describe("markdown_compiler", function () {
 
-  var compiler = require("../markdown");
+  var markdown = require("../markdown");
```
Comment (brainshave) [inline spec/markdown_spec.js]: I think `markdown_setup` would be a better name here. (similar naming is used elsewhere: https://github.com/brainshave/zetzer/blob/master/spec/dot_spec.js#L4)

**Comment by DanielLarsenNZ** [comment]
@brainshave - further tests have proved that marked has GFM and Tables enabled by default. Sorry for wasting your time.

**Pull request closed without merging**